### ISSUE TITLE
Pretrained Model Parameters Do Not Match the Model Built from base.yaml

### ISSUE THREAD

**Issue by chenttt2001** [Zhimin-C/Bridge3D#8]
Hello, author! I am encountering an issue where the pretrained model parameters I downloaded do not match the model constructed using the base.yaml configuration file.

**Comment by Zhimin-C** [comment]
When publishing the code to GitHub, I deleted some unused parts. That is why the parameters do not match.

**Comment by chenttt2001** [comment]
Okay, I have adjusted the parameters in base.yaml so that most of the model's structure matches the parameter file, but there are still some MLP parameters that I couldn't find in the pretrained weights, as shown in the image below. Could you please share the pretrained weights that include these parameters?
![Snipaste_2024-10-28_13-28-45](https://github.com/user-attachments/assets/fae9ca28-3fe1-443e-b7f3-dd5949f63352)

**Comment by Zhimin-C** [comment]
In the fine-tuning stage of downstream tasks, only the plain transformer structure is used. Hence, the pre-trained weights of those parameters can be ignored in the fine-tuning stage.

**Comment by chenttt2001** [comment]
Well, thank you for your help

**Comment by chenttt2001** [comment]
I'm sorry to bother you again. Can you share the weights with these parameters? Maybe my research direction needs them to make some attemptsattempts

**Comment by Zhimin-C** [comment]
Those parameters are not trained by loss and thus are just random numbers.

**Comment by chenttt2001** [comment]
Does that include img_proj?

**Comment by chenttt2001** [comment]
And why are the 3dter parameters I download at **3.Bridge3D Weights** the same as pointmae weights?

**Comment by Zhimin-C** [comment]
I have corrected the link in the readme file. Now you can download the pre-trained 3dter parameters and the correct pre-training weight. Sorry about that.

**Comment by Zhimin-C** [comment]
you can now check the parameters in the pre-trained encoder. I just realized that the link in the README file was incorrect after the last update... Thanks for pointing out!

**Comment by chenttt2001** [comment]
You're welcome. Glad I could help
